src/components/Permissions/PermissionsTab.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import Page from 'components/Page'
+import { useI18n } from 'cozy-ui/transpiled/react'
+import { Tabs, Tab } from 'cozy-ui/transpiled/react/MuiTabs'
+import AppList from './AppList'
+import DataList from './DataList'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import { routes } from 'constants/routes'
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  }
+}
+
+const PermissionsTab = () => {
+  const { t } = useI18n()
+  const [tab, setTab] = useState('apps')
```

Comment:
You should read the value from the router. Because here, you duplicate values from props to state but you also lose the information of the first URL I open. If I click on foo.cozy.cloud/permissions/data, the tab 'apps' will be selected but I don't want that, I want the data tabs. 

The handleChange method should not call a setState but should rather call an history.push()

Reply:
I don't understand which component I should use